Libplanet.Action.Tests/Mocks/MockAccount.cs

```diff
@@ -0,0 +1,121 @@
+using System.Numerics;
+using Libplanet.Action.State;
+using Libplanet.Crypto;
+using Libplanet.Store.Trie;
+using Libplanet.Types.Assets;
+using Libplanet.Types.Tx;
+
+namespace Libplanet.Action.Tests.Mocks
+{
+    public static class MockAccount
```

Comment:
Welp, leave I'll let it slide for now. We might need to get rid of this entirely or overhaul the whole thing.